metrix-explorer-ui/plugins/websocket.js

```diff
@@ -3,8 +3,9 @@ import io from 'socket.io-client'
 
 export default function ({ app }) {
   let ws = io(
-    'ws:' + process.env.metrixinfoWSBase,
-    { transports: ['websocket'] }
+    process.env.hostprot + '//' + process.env.metrixinfoWSBase,
+    //'ws:' + process.env.metrixinfoWSBase,
+    //{ transports: ['websocket'] }
```

Comment:
This is defaulting to port 80... or `ws://` ... How to use `wss://` ?

Reply:
This probably needs its own var for the prefix depending if you want to use SSL or not.
using wss: is the only change from the original code to enable secure comms using the websocket.
process.env.hostprot is https according to the env file, so not sure how swapping ws to https will work.

Reply:
It was force using `ws` for some reason.  I could not get `wss` to work when using ssl.

I got it to work using `https` though..  In the `.env` I declare the protocol to use as `https:`
![image](https://user-images.githubusercontent.com/1578496/135828763-11a286b8-cff3-47d5-9967-1ac3082e3399.png)